src/Compatibility/Microsoft.DotNet.PackageValidation/ValidatePackage.cs

```diff
@@ -32,17 +31,15 @@ protected override void ExecuteCore()
             }
 
             Package package = NupkgParser.CreatePackage(PackageTargetPath, runtimeGraph);
-            PackageValidationLogger log = new();
-
-            new CompatibleTfmValidator(NoWarn, null, RunApiCompat, log).Validate(package);
+            PackageValidationLogger logger = new(Log);
 
+            new CompatibleTfmValidator(NoWarn, null, RunApiCompat, logger).Validate(package);
+            new CompatibleFrameworkInPackageValidator(NoWarn, null, logger).Validate(package);
```

Comment:
It seems like a weird design to have objects that are instantiated but not cached and not used after it's creation. Should these instead be static methods?